recipes/openblas/bld.bat

```diff
@@ -0,0 +1,11 @@
+
+:: Set $HOME to the current dir so msys runs here
+set HOME=%cd%
+
+:: Configure, build, test, and install using `make`.
+bash -lc "make FC=gfortran DYNAMIC_ARCH=1 BINARY=$ARCH NO_LAPACK=0 NO_AFFINITY=1 USE_THREAD=1 PREFIX=`echo $LIBRARY_PREFIX | tr '\\' '/'`"
```

Comment:
> Do we know if openblas is going to be supporting msvc better in the future?

It's more like, is MSVC going to grow the features needed to support openblas better, and that's not going to happen. Maybe clang + the mythical [fortran LLVM frontend](https://www.llnl.gov/news/nnsa-national-labs-team-nvidia-develop-open-source-fortran-compiler-technology) will eventually become an MSVC-compatible alternative, but until then it's gcc+mingw-w64 based toolchains or nothing.